concepticondata/conceptlists/Steinthal-1875-1565.tsv

```diff
@@ -0,0 +1,1566 @@
+Steinthal-1875-1565-357	357	Schwieger-Vater, -Mutter, -Sohn, -Tochter	Schwiegermutter	mother-in-law	1050	MOTHER-IN-LAW
+Steinthal-1875-1565-358	358	Schwieger-Vater, -Mutter, -Sohn, -Tochter	Schwiegersohn	son-in-law	1056	SON-IN-LAW
+Steinthal-1875-1565-359	359	Schwieger-Vater, -Mutter, -Sohn, -Tochter	Schwiegertochter	daughter-in-law	1057	DAUGHTER-IN-LAW
+Steinthal-1875-1565-360	360	Schwager, Schwägerin (meiner Frau Bruder, meiner Schwester Mann; meines Bruders Frau; nennt eine Frau ihren Schwager und ihre Schwägerin ebenso? also ihres Mannes Bruder, ihrer Schwester Mann, ihres Bruders Frau)	Schwager	brother-in-law	2314	BROTHER-IN-LAW
+Steinthal-1875-1565-361	361	Schwager, Schwägerin (meiner Frau Bruder, meiner Schwester Mann; meines Bruders Frau; nennt eine Frau ihren Schwager und ihre Schwägerin ebenso? also ihres Mannes Bruder, ihrer Schwester Mann, ihres Bruders Frau)	Schwägerin	sister-in-law	2317	SISTER-IN-LAW
+Steinthal-1875-1565-362	362	Schwager, Schwägerin (meiner Frau Bruder, meiner Schwester Mann; meines Bruders Frau; nennt eine Frau ihren Schwager und ihre Schwägerin ebenso? also ihres Mannes Bruder, ihrer Schwester Mann, ihres Bruders Frau)	Schwester der eigenen Frau	sister of one's wife		
+Steinthal-1875-1565-363	363	Schwager, Schwägerin (meiner Frau Bruder, meiner Schwester Mann; meines Bruders Frau; nennt eine Frau ihren Schwager und ihre Schwägerin ebenso? also ihres Mannes Bruder, ihrer Schwester Mann, ihres Bruders Frau)	Schwester des eigenen Mannes	sister of one's husband		
+Steinthal-1875-1565-364	364	Schwager, Schwägerin (meiner Frau Bruder, meiner Schwester Mann; meines Bruders Frau; nennt eine Frau ihren Schwager und ihre Schwägerin ebenso? also ihres Mannes Bruder, ihrer Schwester Mann, ihres Bruders Frau)	Bruder der eigenen Frau	brother of one's wife	2380	BROTHER-IN-LAW (OF MAN)
+Steinthal-1875-1565-365	365	Schwager, Schwägerin (meiner Frau Bruder, meiner Schwester Mann; meines Bruders Frau; nennt eine Frau ihren Schwager und ihre Schwägerin ebenso? also ihres Mannes Bruder, ihrer Schwester Mann, ihres Bruders Frau)	Bruder des eigenen Mannes	brother of one's husband		
+Steinthal-1875-1565-366	366	Schwager, Schwägerin (meiner Frau Bruder, meiner Schwester Mann; meines Bruders Frau; nennt eine Frau ihren Schwager und ihre Schwägerin ebenso? also ihres Mannes Bruder, ihrer Schwester Mann, ihres Bruders Frau)	Mann der eigenen Schwester	husband of one's sister		
+Steinthal-1875-1565-367	367	Schwager, Schwägerin (meiner Frau Bruder, meiner Schwester Mann; meines Bruders Frau; nennt eine Frau ihren Schwager und ihre Schwägerin ebenso? also ihres Mannes Bruder, ihrer Schwester Mann, ihres Bruders Frau)	Frau des eigenen Bruders	wife of one's brother		
+Steinthal-1875-1565-368	368	Greis	Greis	old man	406	OLD MAN
+Steinthal-1875-1565-369	369	alte Frau	alte Frau	old woman	907	OLD WOMAN
+Steinthal-1875-1565-370	370	Er ist alt.	Er ist alt.	He is old.		
```

Comment:
Not 2112?

Reply:
I was hesitant because it is a sentence. But since we do not have another OLD (AGED)  in the list, I guess this could be mapped.